integration/integration_test.go

```diff
@@ -72,8 +74,69 @@ func TestRunWithKind(t *testing.T) {
 			expectedData := strings.TrimSpace(string(c))
 			resultData = strings.TrimSpace(resultData)
 			if expectedData != resultData {
-				t.Errorf("expected: %q\n\n Got %q\n\n", expectedData, resultData)
+				t.Errorf("expected results\n\nExpected\t(<)\nResult\t(>)\n\n%s\n\n", generateDiff(expectedData, resultData))
 			}
 		})
 	}
 }
+
+func generateDiff(source, target string) string {
```

Comment:
I would add a comment to the effect that this is a simplistic diff that will be thrown off by the addition or removal of lines. (Full diff is pretty hard! I'd have used a library or shelled out to `diff` to get this done quickly. I'm OK with this approach though as it will help a lot in the normal scenario where an individual test gives us the wrong output.)